wiki/Tournaments/GTS/KGTS_2019/en.md

```diff
@@ -0,0 +1,284 @@
+---
+tags:
+- KGTS 2019
+- KGTS2019
+---
+
+# Katacheh Global Taiko Showdown 2019
+
+![KGTS 2019 Logo](img/logo.jpg)
+
+The **Katacheh Global Taiko Showdown 2019** (***KGTS 2019***) was a worldwide  double-elimination one-on-one osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_CA] [DuckyDoom](https://osu.ppy.sh/users/3153062), and ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133). The mappool used in the tournament includes **[katacheh](https://osu.ppy.sh/beatmapsets?m=1&s=any&q=katacheh) maps** only. It was the first installment of the Katacheh Global Taiko Showdown and part of the Global Taiko Showdown series.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-08-27/2019-09-15 |
```

Comment:
same as other tournaments, don't title case stuff unless if it's actually a title (pretty much nothing is). table headers, table entries, and section headers should all be sentence case

Reply:
+ match results header, some of the Positions for staff member